app.js

```diff
@@ -0,0 +1,462 @@
+valueLoad() // loading entered size of the board
+class board {
+    tableId = []
+    buttonId = []
+    btnId = 0
+    table = document.querySelector('table')
+    x
+    y
+    constructor(x,y) {
+        this.creatingTable(x,y)
+        this.x = x              // returning size of the board
+        this.y = y
+        this.size = x*y
+
+    }
+    creatingTable(x,y){
+        for (let i = 0; i < y; i++) {
+            let tr = document.createElement('tr'); // creating rows for the table
+
+            for (let j = 0; j < x; j++) {
+                let td = document.createElement('td'); // and cells
+                tr.appendChild(td);
+                td.style.border = '1px solid black'
+                td.innerHTML = ''
+                td.style.width = '55px'
+                td.style.height = '55px'
+
+                let btn = document.createElement('button') // for each cell was created a button
+                document.body.appendChild(btn)
+                btn.style.width = '60px'
+                btn.style.height = '60px'
+                btn.style.marginLeft = '-4.5em'
+
+
+                let btnStrId = this.btnId.toString() //giving an id to each button
+                btn.id = btnStrId
+                this.buttonId.push(btn.id)
+                let col = document.getElementById(btn.id)
+                col.style.background = "grey"
+
+                let tdId = this.btnId + 'td' // giving an id to each element of the row
+                td.id = tdId
+                this.tableId.push(td.id)
+                tr.appendChild(btn)
+                this.btnId++
+
+
+            }
+            this.table.appendChild(tr);
+        }
```

Comment:
Board should contain list of fields. Board should have render method which will invoke render on each field. Please create class diagram this way you will understand connections between classes. Please remember about SOLID principle when you are designing class diagram and most important single responsibility of each class. Change the whole code according to this comment.